backend/src/common/utils/userHelper.ts

```diff
@@ -0,0 +1,20 @@
+import { createToken } from './tokenHelper';
+import { IUser } from '../models/user/IUser';
+import { IUserResponse } from '../models/user/IUserResponse';
+import { IRegisterUser } from '../models/user/IRegisterUser';
+import { ICreateUser } from '../models/user/ICreateUser';
+
+export const signUpResponseMapper = (user: IUser): IUserResponse => {
+  const responseUser = user;
+  delete responseUser.password;
```

Comment:
maybe not best solution, but `const { password, ...responseUser } = user` causes linter error.